plugins/janus_videoroom.c

```diff
@@ -3343,6 +3374,9 @@ static void *janus_videoroom_handler(void *data) {
 				json_object_set_new(event, "id", json_integer(user_id));
 				json_object_set_new(event, "private_id", json_integer(publisher->pvt_id));
 				json_object_set_new(event, "publishers", list);
+				/* Notify all participants the publisher has joined the room */
```

Comment:
"We may need to notify all participants that a new user joined the room"

Alternate text since the current comment is broken, as it seems to indicate that: 1) you're notifying about a publisher, which isn't true (no she/he's publishing anything yet), and 2) that this notification will always happen (when that method actually checks the flag to see if it will do that).